client/src/components/Guests.jsx

```diff
@@ -33,40 +34,87 @@ class Guests extends React.Component {
     this.setState({
       guestColor: color,
       guestText: textColor,
-      guestArrow: arrowDir
+      guestArrow: arrowDir,
+      guestPadding: padding,
+      infantPadding: babyPadding
     })
   }
 
   showGuestsPanel() {
     let show = this.state.showPanel ? false : true;
     let bottom = this.state.borderBottom === '1px solid #e4e4e4' ? '2px solid #007A87' : '1px solid #e4e4e4';
+    let height = this.state.boxHeight === '42px' ? '50px' : '42px';
+
     this.setState({
       showPanel: show,
-      borderBottom: bottom
+      borderBottom: bottom,
+      boxHeight: height
     })
   }
 
   render() {
+    let guestInfantInfo = this.props.house.infant_guest_eligible ? 'count' : 'don\'t count';
+    let guestSyntax = this.props.total === 1 ? 'guest' : 'guests';
+    let infantSyntax;
+
+    if (!this.props.house.infant_guest_eligible) {
+      if (this.props.infants > 1) {
+        infantSyntax = ', ' + this.props.infants + ' infants';
+      } else if (this.props.infants === 1) {
+        infantSyntax = ', ' + this.props.infants + ' infant';
+      }
+    }
+
     let guestsPanel = (
```

Comment:
There is a good amount of repetitive code here. Personally, I think it looks great. However, in the words of Josh, Jordan, and Ram, repetitive code is bad. I think there is a way to condense this, but It most definitely is not a priority. MVP MVP MVP!